### PR TITLE
[WIP] - ROC-6304-Hearing-Location-Serialization-Issue

### PR DESCRIPTION
**Pull request by karthikkannankainos**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-6304

### Change description
Hearing Location serialization issue.

### CODE REVIEW

**Comment by robertparkinson** [comment]
Sonar may need you to put in more tests around the new functions you've created.

File: src/main/app/claims/models/directions-questionnaire/directionsQuestionnaire.ts [hunk 1]
```diff
@@ -67,6 +56,40 @@ export namespace DirectionsQuestionnaire {
     }
   }
 
+
+  export function toHearingLocation(directionsQuestionnaire: DirectionsQuestionnaireDraft): HearingLocation {
```
Comment (KiranVarma02) [inline src/main/app/claims/models/directions-questionnaire/directionsQuestionnaire.ts]: maybe we don't need to export function, i don't think we can or will use it anywhere else...

File: src/main/app/claims/models/directions-questionnaire/directionsQuestionnaire.ts [hunk 2]
```diff
@@ -67,6 +56,40 @@ export namespace DirectionsQuestionnaire {
...
+        directionsQuestionnaire.hearingLocation.courtName : directionsQuestionnaire.hearingLocation.alternativeCourtName,
+      hearingLocationSlug: (directionsQuestionnaire.hearingLocationSlug && directionsQuestionnaire.hearingLocationSlug.length) ? directionsQuestionnaire.hearingLocationSlug : undefined,
+      courtAddress: undefined,
+      locationOption: toLocationOption(directionsQuestionnaire.hearingLocation),
+      exceptionalCircumstancesReason: directionsQuestionnaire.exceptionalCircumstances ?
+        directionsQuestionnaire.exceptionalCircumstances.reason : undefined
+    }
+  }
+
+  export function toLocationOption(hearingLocation: DraftHearingLocation): CourtLocationType {
```
Comment (KiranVarma02) [inline src/main/app/claims/models/directions-questionnaire/directionsQuestionnaire.ts]: maybe we don't need to export function, i don't think we can or will use it anywhere else...

File: src/main/app/claims/models/directions-questionnaire/directionsQuestionnaire.ts (hunk 1 above)
Comment (karthikkannankainos) [reply src/main/app/claims/models/directions-questionnaire/directionsQuestionnaire.ts]: Have reverted. @KiranVarma02

File: src/main/app/claims/models/directions-questionnaire/directionsQuestionnaire.ts (hunk 2 above)
Comment (karthikkannankainos) [reply src/main/app/claims/models/directions-questionnaire/directionsQuestionnaire.ts]: done

**Pull request merged**